tools/spatyper/spatyper.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="spatyper" name="Generate spa type identification" version="@VERSION@+galaxy0" profile="21.01">
+    <macros>
+        <token name="@VERSION@">0.3.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">spatyper</requirement>
+    </requirements>
+    <command detect_errors="aggressive"> <![CDATA[
+        spaTyper -f '${input1}' -r '${input2}' > '${output1}' 
+    ]]> </command>
+    <inputs>
+        <param type="data" name="input1" format="fasta" label="SPA Fasta file" help="Staphylococcus aureus - staphylococcal protein A fasta file. This will be assigned a type based on its repeat region."/>
+        <param type="data" name="input2" format="fasta" label="Repeats Library Multifasta" help="A multifasta representing a set of known repeat regions found in staphylococcal protein A. This can be downloaded at (http://spa.ridom.de/dynamic/sparepeats.fasta)"/>
+    </inputs>
+    <outputs>
+        <data name="output1" format="txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="input.fasta"/>
+            <param name="input2" value="sparepeats.fasta"/>
+            <output name="output1">
+                <assert_contents>
+                    <has_text text = "t11215" />
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+
+USAGE: spaTyper [-h] [-r REPEAT_FILE] [-o REPEAT_ORDER_FILE] [-d FOLDER] [-f FASTA [FASTA ...]] [-g GLOB] [-e] [--output OUTPUT] [--version] [--debug] [--info]
```

Comment:
The help documentation is written using reStructuredText rather than markdown. This means that blocks of text that should not be formatted (like the usage statement here) need to be preceded by a double colon `::` and indented. For example:

```
::

    Usage statement
    Second part of usage

Outside of usage statement.
```

See https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-help for information on the Galaxy `help` element and https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks for documentation on literal blocks in reStructuredText.

You may also want to include an example of the output file here that someone should expect to see by the tool (essentially a copy of the `output.txt` file you have provided).

Reply:
Should be resolved by most recent changes